### PR TITLE
test=develop, update protobuf

### PR DESCRIPTION
**Pull request by JiabinYang**

### CODE REVIEW

**Comment by luotao1** [comment]
https://github.com/PaddlePaddle/Paddle/blob/4c8feae43d226646397f812b945a4b5b07e5a6f9/cmake/external/python.cmake#L77-L80
这里要改成3.6

File: python/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 requests==2.9.2
 numpy>=1.12
-protobuf==3.1
+protobuf>=3.6
```
Comment (luotao1) [inline python/requirements.txt:3]: >=还是==? >=没有验证过。

Comment (JiabinYang) [reply python/requirements.txt:3]: 验证了下可以用>=

**Comment by JiabinYang** [comment]
> [Paddle/cmake/external/python.cmake](https://github.com/PaddlePaddle/Paddle/blob/4c8feae43d226646397f812b945a4b5b07e5a6f9/cmake/external/python.cmake#L77-L80)
> Lines 77 to 80 in [4c8feae](/PaddlePaddle/Paddle/commit/4c8feae43d226646397f812b945a4b5b07e5a6f9)
>  IF(${PY_GOOGLE.PROTOBUF_VERSION} AND ${PY_GOOGLE.PROTOBUF_VERSION} VERSION_LESS "3.0.0") 
>      MESSAGE(FATAL_ERROR "Found Python Protobuf ${PY_GOOGLE.PROTOBUF_VERSION} < 3.0.0, " 
>      "please use pip to upgrade protobuf. pip install -U protobuf") 
>  ENDIF() 
> 
> 这里要改成3.6

done

**Review by luotao1 — APPROVED** [review]
LGTM!

**Pull request merged**